src/js/common/schemaform/review/ReadOnlyArrayField.jsx

```diff
@@ -0,0 +1,92 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {
+  toIdSchema,
+  deepEquals
+} from 'react-jsonschema-form/lib/utils';
+
+class BasicArrayField extends React.Component {
+  shouldComponentUpdate = (nextProps) => {
+    return !deepEquals(this.props, nextProps);
+  }
+
+  getItemSchema = (index) => {
+    return this.props.schema.items[index];
+  }
+
+  render() {
+    const {
+      uiSchema,
+      errorSchema,
+      idSchema,
+      formData,
+      disabled,
+      readonly,
+      registry
+    } = this.props;
+    const definitions = registry.definitions;
+    const { SchemaField } = registry.fields;
+    const uiOptions = uiSchema['ui:options'] || {};
+
+    const items = formData || [];
+
+    return (
+      <div className="schemaform-field-container">
+        <div>
+          {items.map((item, index) => {
+            const itemSchema = this.getItemSchema(index);
+            const itemIdPrefix = `${idSchema.$id}_${index}`;
+            const itemIdSchema = toIdSchema(itemSchema, itemIdPrefix, definitions);
+
+            return (
+              <div key={index} className="va-growable-background">
```

Comment:
Just want to make sure using index as key here won't be an issue as items won't be re-orderable, etc.